src/js/quickview.ts

```diff
@@ -42,5 +42,8 @@ export default function initQuickviews() {
       });
       event.preventDefault();
     });
+    prestashop.on('updateCart', () => {
+      $(selectorsMap.quickviewModal).modal('hide');
```

Comment:
jQuery in hummingbird?
Someone really has problem with this :)

Reply:
Oops, sorry @mparvazi

Reply:
so the plan is for entire quickview feature to be reworked before the release?

Reply:
> Oops, sorry @mparvazi

@Hlavtox I was just kidding ;)
You're the best  :100: